src/pks/transport/transport_ats_pk.cc

```diff
@@ -465,6 +477,31 @@ void Transport_ATS::Initialize(const Teuchos::Ptr<State>& S)
       }
     }
 
+
+    // Teuchos::ParameterList& conc_sources_list = plist_->sublist("source terms").sublist("component concentration source");
+    // for (const auto& it : conc_sources_list) {
+    //   std::string name = it.first;
+    //   if (conc_sources_list.isSublist(name)) {
+    //     Teuchos::ParameterList& src_list = conc_sources_list.sublist(name);
+    //     std::string src_type = src_list.get<std::string>("spatial distribution method", "none");
+
+    //       Teuchos::RCP<TransportDomainFunction> src =
+    //           factory.Create(src_list, "source function", AmanziMesh::CELL, Kxy);
+
+    //       std::vector<std::string> tcc_names = src_list.get<Teuchos::Array<std::string>>("component names").toVector();
+    //       src->set_tcc_names(tcc_names);
+
+    //       // set the component indicies
+    //       for (const auto& n : src->tcc_names()) {
+    //         src->tcc_index().push_back(FindComponentNumber(n));
+    //       }
+
+    //       src->set_state(S_);
+    //       srcs_.push_back(src);
+    //     }
+    //   }
+    // }
```

Comment:
Can this be removed?  Or is it newly added code for the new source type?

Reply:
This is the pace for new code for the new source type.